lib/cron.dart

```diff
@@ -43,6 +45,15 @@ class Schedule {
   /// The weekdays a Task should be started.
   final List<int>? weekdays;
 
+  bool runAt(DateTime time) {
```

Comment:
Does this needs to be public?

Reply:
I'm using this method in my project, so I'd prefer if it were.

Reply:
I don't mind having it public, but let's add a small documentation describing what it means.

Also, really nitpicking, but the current name suggest a command: [let's] run at... Instead, maybe `willRunAt`, `shouldRunAt` or `runsAt` seems to be more fitting, which indicates that the method returns with a yes/no (bool) answer. I don't have a strong preference here, but I think a different name would really help the readability.